docs/UserGuide.md

```diff
@@ -163,15 +163,23 @@ Parameters:
 
 <div markdown="span" class="alert alert-primary">
   :exclamation: <b>Important:</b>
-Some commands are not available while on profile view page. 
+Only specific commands are available in the profile view page. 
 The list of available commands are: 
 <ul>
   <li> <code>help</code> </li>
-  <li> <code>edit</code> </li>
+  <li> <code>save</code> </li>
   <li> <code>exit</code> command to <b>exit the profile view page</b> </li>
 </ul>
 </div> 
 
+Editing a fosterer’s details in profile view page:
+
+1. Enter a fosterer’s profile view page with `view INDEX` 
+2. Type in the name of the field you want to edit onto the command box, for example name. This will make the textbox visible next to the name field and automatically set the text cursor to the textbox.
+3. Edit the original value - if it exists - to a new value, for example John.
+4. Press enter to return the text cursor back to the command box.
+5. Type in save  command and enter to save the changes of the fosterer. 
```

Comment:
very minor change but because I am comparing this to the ug doc now, do you wanna do like `name` , `John` here too to standardise? not very urgent, just a suggestion